### DISCUSSION TITLE
关于vux3.0的一些设想

### DISCUSSION THREAD

**Discussion by eJayYoung** [airyland/vux#3807] [Ideas]
1、支持vue3
2、支持vite

@airyland @caozhong1996

**Comment by caozhong1996** [comment]
人太少了，要摇人才行

**Reply by eJayYoung** [reply]
> 人太少了，要摇人才行

可以慢慢开始筹划呗，反正也是一个锻炼，我想了下，这个仓库的owner其实基本上不参与维护了，迟早是需要Duplicate的，可以新建一个vux的组织，然后新建一个vux-next的仓库吧，我觉得vux的核心其实主要还是在于基于weui的开箱即用的方案，你有什么好的想法也可以一起讨论

**Reply by eJayYoung** [reply]
貌似已经有个vuxjs的组织了，https://github.com/vuxjs
不过也没人管了

**Comment by eJayYoung** [comment]
## vux-next 
repo: https://github.com/vux-next/vux-next

design: https://github.com/weui/weui-design

Tech Stack：

- typescript（Type Definition）
- vue@next（Core Runtime）
- vite（Development & Deploy）
- lerna + yarn workspace（Package Management）
- css modules or css-in-js（CSS Not Decided）

Package：

- vux
- vux-docs
- vux-playground

WorkFlow：

- eslint + prettier + lint-staged
- Commitizen + conventional-changelog


@caozhong1996 这是我的一些规划，可以看看有啥补充的

**Reply by eJayYoung** [reply]
目前参考的是
https://github.com/TuSimple/naive-ui

**Reply by caozhong1996** [reply]
lerna + yarn workspace（Package Management） 感觉可以用 pnpm 的 workspace，最近用了一下还挺好用的，很多主流库都迁到pnpm 了

**Reply by caozhong1996** [reply]
vux 有一个 v3 分支，之前搭了一套脚手架，但是没什么时间搞后续了

**Reply by caozhong1996** [reply]
加我微信联系吧，我把作者的微信推给你，看能不能让他也给你开下权限

**Reply by eJayYoung** [reply]
> lerna + yarn workspace（Package Management） 感觉可以用 pnpm 的 workspace，最近用了一下还挺好用的，很多主流库都迁到pnpm 了

嗯，我看naive-ui也是用的pnpm，可以尝试一下
